src/routes/Schedule.svelte

```diff
@@ -1,15 +1,9 @@
 <script lang="ts">
 	import InfoBox from '$lib/layout/InfoBox.svelte';
-	import PersonLink from '$lib/layout/PersonLink.svelte';
 </script>
 
 <InfoBox title="Friday schedule">
-	<p>
-		JSCraftCamp 2024 will be facilitated by <PersonLink
-			linkedIn="https://www.linkedin.com/in/stefanie-hasler-05458a4/"
-			mastodon="">Stefanie Hasler</PersonLink
-		>!
-	</p>
+	<p>JSCraftCamp 2025 will be facilitated by ... to be announced!</p>
```

Comment:
Can we remove the schedule. In order to reduce the content on the page.

Reply:
We'll keep the schedule and put it waaaaaaay down on the page.